### ISSUE TITLE
Maven Test Error

### ISSUE THREAD

**Issue by geekdinazor** [marytts/marytts#381]
When i compile latest GIT code on Xubuntu 15.04 with maven it gives error. [Output of mvn install -X](http://www.sudrap.org/paste/text/610328/)

Surefire report Output:
 ```
-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 43, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 9.253 sec <<< FAILURE! - in TestSuite
testExpandDate(marytts.language.en.PreprocessTest)  Time elapsed: 0.604 sec  <<< FAİLURE!
java.text.ParseException: Unparseable date: "06/29/1993"
	at com.ibm.icu.text.DateFormat.parse(DateFormat.java:667)
	at marytts.language.en.Preprocess.expandDate(Preprocess.java:583)
	at marytts.language.en.PreprocessTest.testExpandDate(PreprocessTest.java:171)

testExpandDate(marytts.language.en.PreprocessTest)  Time elapsed: 0.002 sec  <<< FAİLURE!
java.text.ParseException: Unparseable date: "06/22/1992"
	at com.ibm.icu.text.DateFormat.parse(DateFormat.java:667)
	at marytts.language.en.Preprocess.expandDate(Preprocess.java:583)
	at marytts.language.en.PreprocessTest.testExpandDate(PreprocessTest.java:171)
```

**Comment by psibre** [comment]
Cannot reproduce. This is what I get:
```
$ mvn test -pl :marytts-lang-en -Dtest=PreprocessTest
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building marytts-lang-en 5.2-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (enforce-java-7-or-higher) @ marytts-lang-en ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.3:create (default) @ marytts-lang-en ---
[INFO] Executing: /bin/sh -c cd /Users/ingmar/workspace/marytts/marytts-languages/marytts-lang-en && git rev-parse --verify HEAD
[INFO] Working directory: /Users/ingmar/workspace/marytts/marytts-languages/marytts-lang-en
[INFO] Storing buildNumber: c13201e3aaa87157cf356e10c892ecbbf442ea60 at timestamp: 1439309880150
[INFO] Storing buildScmBranch: master
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ marytts-lang-en ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 31 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ marytts-lang-en ---
[INFO] Changes detected - recompiling the module!
[INFO] Using Groovy-Eclipse compiler to compile both Java and Groovy files
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ marytts-lang-en ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 4 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ marytts-lang-en ---
[INFO] Changes detected - recompiling the module!
[INFO] Using Groovy-Eclipse compiler to compile both Java and Groovy files
[INFO] 
[INFO] --- maven-surefire-plugin:2.17:test (default-test) @ marytts-lang-en ---
[INFO] Surefire report directory: /Users/ingmar/workspace/marytts/marytts-languages/marytts-lang-en/target/surefire-reports

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=128m; support was removed in 8.0
Running marytts.language.en.PreprocessTest
Configuring TestNG with: TestNG652Configurator
INFO  main: Mary starting up...
INFO  main: Specification version 5.2-SNAPSHOT
INFO  main: Implementation version unknown
INFO  main: Running on a Java 1.8.0_51 implementation by Oracle Corporation, on a Mac OS X platform (x86_64, 10.9.5)
INFO  ModuleRegistry: Now initiating mary module 'marytts.language.en.JTokeniser'
INFO  ModuleRegistry: Now initiating mary module 'marytts.language.en.Preprocess'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.JPhonemiser(en_US.)'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.JPhonemiser(en_GB.)'
INFO  ModuleRegistry: Now initiating mary module 'marytts.language.en.Prosody'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.SimplePhoneme2AP(en_US)'
INFO  ModuleRegistry: Now initiating mary module 'marytts.language.en.PronunciationModel'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.OpenNLPPosTagger(en,en.pos)'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.Synthesis'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.TextToMaryXML'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.SableParser'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.SSMLParser'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.APMLParser'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.EmotionmlParser'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.JTokeniser'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.DummyTokens2Words'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.ProsodyGeneric'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.PronunciationModel'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.TargetFeatureLister'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.HalfPhoneTargetFeatureLister'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.AcousticModeller'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.RealisedAcoustparamsExtractor'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.RealisedDurationsExtractor'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.HTSEngine'
INFO  ModuleRegistry: Now initiating mary module 'marytts.modules.PraatTextGridGenerator'
INFO  JTokeniser: Module started (RAWMARYXML->TOKENS, locale en).
INFO  JTok: loading language resources for en from jtok/en
INFO  JTokeniser: Starting power-on self test.
INFO  JTokeniser: Power-on self test complete.
INFO  Preprocess: Module started (TOKENS->WORDS, locale en).
INFO  Preprocess: Starting power-on self test.
INFO  Preprocess: Power-on self test complete.
INFO  JPhonemiser: Module started (PARTSOFSPEECH->PHONEMES, locale en_US).
INFO  JPhonemiser: Starting power-on self test.
INFO  JPhonemiser: Power-on self test complete.
INFO  JPhonemiser: Module started (PARTSOFSPEECH->PHONEMES, locale en_GB).
INFO  JPhonemiser: Starting power-on self test.
INFO  JPhonemiser: Power-on self test complete.
INFO  Prosody: Module started (PHONEMES->INTONATION, locale en).
INFO  Prosody: Starting power-on self test.
INFO  Prosody: Power-on self test complete.
INFO  SimplePhoneme2AP: Module started (SIMPLEPHONEMES->ACOUSTPARAMS, locale en_US).
INFO  SimplePhoneme2AP: Starting power-on self test.
INFO  SimplePhoneme2AP: Power-on self test complete.
INFO  PronunciationModel: Module started (INTONATION->ALLOPHONES, locale en).
INFO  PronunciationModel: Starting power-on self test.
INFO  PronunciationModel: Power-on self test complete.
INFO  OpenNLPPosTagger: Module started (WORDS->PARTSOFSPEECH, locale en).
INFO  OpenNLPPosTagger: Starting power-on self test.
INFO  OpenNLPPosTagger: Power-on self test complete.
INFO  UnitSelectionSynthesizer: started.
INFO  TargetFeatureLister: Module started (ACOUSTPARAMS->TARGETFEATURES, locale null).
INFO  HTSEngine: Module started (TARGETFEATURES->AUDIO, locale null).
INFO  HMMSynthesizer: started.
INFO  Synthesis: Module started (ACOUSTPARAMS->AUDIO, locale null).
INFO  HMMSynthesizer: Starting power-on self test.
INFO  TextToMaryXML: Module started (TEXT->RAWMARYXML, locale null).
INFO  TextToMaryXML: Starting power-on self test.
WARN  TextToMaryXML: Locale is null, overriding with en_US
WARN  TextToMaryXML: Locale is null, overriding with en_US
INFO  TextToMaryXML: Power-on self test complete.
INFO  SableParser: Module started (SABLE->RAWMARYXML, locale null).
INFO  SableParser: Starting power-on self test.
INFO  SableParser: Power-on self test complete.
INFO  SSMLParser: Module started (SSML->RAWMARYXML, locale null).
INFO  SSMLParser: Starting power-on self test.
INFO  SSMLParser: Power-on self test complete.
INFO  APMLParser: Module started (APML->RAWMARYXML, locale null).
INFO  APMLParser: Starting power-on self test.
INFO  APMLParser: Power-on self test complete.
INFO  EmotionmlParser: Module started (EMOTIONML->RAWMARYXML, locale null).
INFO  EmotionmlParser: Starting power-on self test.
INFO  EmotionmlParser: Power-on self test complete.
INFO  JTokeniser: Module started (RAWMARYXML->TOKENS, locale null).
INFO  JTok: loading language resources for en from jtok/en
INFO  JTokeniser: Starting power-on self test.
INFO  JTokeniser: Power-on self test complete.
INFO  DummyTokens2Words: Module started (TOKENS->WORDS, locale null).
INFO  DummyTokens2Words: Starting power-on self test.
INFO  DummyTokens2Words: Power-on self test complete.
INFO  Prosody: Module started (PHONEMES->INTONATION, locale null).
INFO  Prosody: Starting power-on self test.
INFO  Prosody: Power-on self test complete.
INFO  PronunciationModel: Module started (INTONATION->ALLOPHONES, locale null).
INFO  PronunciationModel: Starting power-on self test.
INFO  PronunciationModel: Power-on self test complete.
INFO  TargetFeatureLister: Starting power-on self test.
INFO  TargetFeatureLister: Power-on self test complete.
INFO  TargetFeatureLister: Module started (ACOUSTPARAMS->HALFPHONE_TARGETFEATURES, locale null).
INFO  TargetFeatureLister: Starting power-on self test.
INFO  TargetFeatureLister: Power-on self test complete.
INFO  AcousticModeller: Module started (ALLOPHONES->ACOUSTPARAMS, locale null).
INFO  AcousticModeller: Starting power-on self test.
INFO  AcousticModeller: Power-on self test complete.
INFO  Realised acoustparams extractor: Module started (AUDIO->REALISED_ACOUSTPARAMS, locale null).
INFO  Realised acoustparams extractor: Starting power-on self test.
INFO  Realised acoustparams extractor: Power-on self test complete.
INFO  Realised durations extractor: Module started (AUDIO->REALISED_DURATIONS, locale null).
INFO  Realised durations extractor: Starting power-on self test.
INFO  Realised durations extractor: Power-on self test complete.
INFO  Praat TextGrid generator: Module started (AUDIO->PRAAT_TEXTGRID, locale null).
INFO  Praat TextGrid generator: Starting power-on self test.
INFO  Praat TextGrid generator: Power-on self test complete.
INFO  main: Startup complete.
WARN  Preprocess: Could not expand unknown abbreviation "qlsv234toinsdcsdl.", ignoring
INFO  R 1: New request (input type "TEXT", output type "WORDS)
INFO  R 1: Handling request using the following modules:
INFO  R 1: - TextToMaryXML (marytts.modules.TextToMaryXML)
INFO  R 1: Next module: TextToMaryXML
INFO  R 1: Handling request using the following modules:
INFO  R 1: - JTokeniser (marytts.language.en.JTokeniser)
INFO  R 1: - Preprocess (marytts.language.en.Preprocess)
INFO  R 1: Next module: JTokeniser
INFO  R 1: Next module: Preprocess
INFO  R 1: Request processed in 6 ms.
INFO  R 1:    TextToMaryXML took 0 ms
INFO  R 1:    JTokeniser took 3 ms
INFO  R 1:    Preprocess took 1 ms
Tests run: 37, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.097 sec - in marytts.language.en.PreprocessTest
INFO  main: Shutting down modules...
INFO  JTokeniser: Module shut down.
INFO  Preprocess: Module shut down.
INFO  JPhonemiser: Module shut down.
INFO  JPhonemiser: Module shut down.
INFO  Prosody: Module shut down.
INFO  SimplePhoneme2AP: Module shut down.
INFO  PronunciationModel: Module shut down.
INFO  OpenNLPPosTagger: Module shut down.
INFO  Synthesis: Module shut down.
INFO  TextToMaryXML: Module shut down.
INFO  SableParser: Module shut down.
INFO  SSMLParser: Module shut down.
INFO  APMLParser: Module shut down.
INFO  EmotionmlParser: Module shut down.
INFO  JTokeniser: Module shut down.
INFO  DummyTokens2Words: Module shut down.
INFO  Prosody: Module shut down.
INFO  PronunciationModel: Module shut down.
INFO  TargetFeatureLister: Module shut down.
INFO  TargetFeatureLister: Module shut down.
INFO  AcousticModeller: Module shut down.
INFO  Realised acoustparams extractor: Module shut down.
INFO  Realised durations extractor: Module shut down.
INFO  HTSEngine: Module shut down.
INFO  Praat TextGrid generator: Module shut down.
INFO  main: Shutdown complete.

Results :

Tests run: 37, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.349 s
[INFO] Finished at: 2015-08-11T17:18:04+01:00
[INFO] Final Memory: 18M/279M
[INFO] ------------------------------------------------------------------------
```
Please note the buildnumber, i.e.,
```
$ git describe
v5.1.2-144-gc13201e
```
> When i compile latest GIT code 

Please be more specific.

**Comment by psibre** [comment]
Come to think of it, this seems to be a duplicate of #349, although I don't yet understand why it fails on your system.

Could you please tell me which system `locale` you're using?

**Comment by geekdinazor** [comment]
My locale is tr_TR.UTF-8.  I remove other language files and run ```LC_ALL=C mvn install``` command runs perfectly.

**Comment by psibre** [comment]
We need to handle this more systematically in the way we rely on ICU4J.

Closing this in favor of #349

**Comment by seblemaguer** [comment]
Actually solved in #421 (#420 is a mistaken merge)
